docs/user-access-token.md

```diff
@@ -80,3 +80,16 @@ Your client id and secret will also vary per environment.
 ## Audience
 
 Both authorization flows require an `audience` parameter when redirecting the user to the authorization server to log in. In both scenarios the audience must be set to  `https://api.publiq.be`.
+
+## Expiration
+
+After you have received your user access token through one of the two OAuth flows described above, there are two ways to check when your token expires:
+
+1.  Cache the `expires_in` property included in the token response, and the time that you requested the token. Using these two parameters, you can calculate the expiration time of the token and request a new one when it is expired. Note that if you follow this approach, you should account for clock skew between your server and the APIs' servers, so it's best to already request a new token a couple of minutes before the cached one will expire.
+2.  Keep using the same cached token until you get a `401` response from an API endpoint, at which point you can request a new token and perform the failed request again with the new token.
+
+## Token parsing
+
+Avoid parsing the token as a JWT, for example to check its expiration time. While publiq's access tokens are JWTs right now, this is not guaranteed to always be the case and it might happen that you get a token that is not a parseable JWT at some point in the future.
+
+Claims inside the token are also not guaranteed to be stable and may change over time.
```

Comment:
idem als bij client access tokens.

ik realiseer mij nu wel dat we niets zeggen over idTokens. en hoewel niet echt nodig, is dit misschien toch een logische plaats om ernaar te verwijzen en de custom claims publiek te documenteren?

Reply:
> ik realiseer mij nu wel dat we niets zeggen over idTokens. en hoewel niet echt nodig, is dit misschien toch een logische plaats om ernaar te verwijzen en de custom claims publiek te documenteren?

Ja goed punt, maar het is misschien ook raar om over id tokens te spreken op een pagina die specifiek over access tokens gaat? In elk geval doe ik het dan in een aparte PR